projects/storefrontlib/src/cms-components/misc/qualtrics/qualtrics.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { QualtricsConfigService } from '@spartacus/core';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'cx-qualtrics',
+  template: `
+    <ng-container *ngIf="qualtricsEnabled$ | async"></ng-container>
+  `,
+})
+export class QualtricsComponent implements OnInit {
+  qualtricsEnabled$: Observable<boolean>;
```

Comment:
we can drop the OnInit and make this 
`qualtricsEnabled$: Observable<boolean> = this.config.load();`